dynamic/src/modules/var.rs

```diff
@@ -113,6 +116,16 @@ pub mod VarModule {
     /// Resets all values
     pub const RESET_ALL:      u8 = 0xFF;
 
+    /// Resets various `VarModule` arrays depending on the mask
```

Comment:
I don't know what will happen when you merge this because I believe I've updated the description of this function
check https://github.com/HDR-Development/HewDraw-Remix/commit/4726c7bbee7a01a3b9ef2f74cdca658d175a3262 for the proper description, fix it if you want or just merge it I don't think it's a big deal